### PR TITLE
Noticias author date

### LINKED ISSUE #136: Missing meta fields on Noticias
Author and Date

### PR DESCRIPTION
**Pull request by mikeyrayvon**
close #136 
add author and date meta fields. order noticias by meta date

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -28,7 +30,7 @@
         </div>
         <div class="article-author font-serif">
           <?php echo __('[:es]Por[:en]By'); ?>:<br/>
-          <?php the_author(); ?>
+          <?php echo !empty($author) ? $author : ''; ?>
```
Comment (mixcas) [inline index.php]: what would come in an empty `$author` that requires this check?

Comment (mixcas) [inline index.php]: i mean, if the author is empty we shouldn't output the whole `.article-author` no?

File: lib/meta-boxes.php [hunk 2]
```diff
@@ -47,6 +47,19 @@ function igv_cmb_metaboxes() {
   ) );
 
   $post_meta->add_field( array(
+    'name' => __( 'Autor', 'cmb2' ),
+    'id'   => $prefix . 'post_author',
+    'type' => 'text',
+  ) );
+
+  $post_meta->add_field( array(
```
Comment (mixcas) [inline lib/meta-boxes.php:55]: If we are using this date to sort the posts too, why not use published date for all of this?

Comment (mikeyrayvon) [reply lib/meta-boxes.php:55]: because editing the published date is weird UX

**Pull request merged**